otherlibs/unix/write_win32.c

```diff
@@ -61,6 +62,48 @@ CAMLprim value caml_unix_write(value fd, value buf, value vofs, value vlen)
   CAMLreturn(Val_long(written));
 }
 
+CAMLprim value caml_unix_write_bigarray(value fd, value vbuf,
+                                        value vofs, value vlen)
+{
+  CAMLparam4(fd, vbuf, vofs, vlen);
+  intnat ofs, len, written;
+  void *buf;
+  DWORD numwritten;
+  DWORD err = 0;
+
+  buf = Caml_ba_data_val(vbuf);
+  ofs = Long_val(vofs);
+  len = Long_val(vlen);
+  written = 0;
+  while (len > 0) {
+    if (Descr_kind_val(fd) == KIND_SOCKET) {
+      int ret;
+      SOCKET s = Socket_val(fd);
+      caml_enter_blocking_section();
+      ret = send(s, buf + ofs, len, 0);
+      if (ret == SOCKET_ERROR) err = WSAGetLastError();
+      caml_leave_blocking_section();
+      if (ret == SOCKET_ERROR && err == WSAEWOULDBLOCK && written > 0) break;
+      numwritten = ret;
+    } else {
+      HANDLE h = Handle_val(fd);
+      caml_enter_blocking_section();
+      if (! WriteFile(h, buf + ofs, len, &numwritten, NULL))
+        err = GetLastError();
+      caml_leave_blocking_section();
+    }
+    if (err) {
+      caml_win32_maperr(err);
+      caml_uerror("write_bigarray", Nothing);
+    }
+    written += numwritten;
+    ofs += numwritten;
+    len -= numwritten;
+  }
+  caml_leave_blocking_section();
```

Comment:
I think this line shouldn't be here, and is causing the remaining appveyor failure.

Reply:
Indeed, fixed. Thanks!